core-bundle/src/Resources/contao/templates/analytics/analytics_google.html5

```diff
@@ -1,16 +1,16 @@
 <?php
 
 // To use this script, please fill in your Google Analytics ID below
-$GoogleAnalyticsId = 'UA-XXXXX-X';
+$GoogleAnalyticsId = 'G-XXXXXXXXXX';
 
 // DO NOT EDIT ANYTHING BELOW THIS LINE UNLESS YOU KNOW WHAT YOU ARE DOING!
-if ('UA-XXXXX-X' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
+if ('G-XXXXXXXXXX' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
 
 <script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-  ga('create', '<?= $GoogleAnalyticsId ?>', 'auto');
-  ga('set', 'anonymizeIp', true);
-  ga('send', 'pageview');
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', '<?= $GoogleAnalyticsId ?>');
```

Comment:
How can this new script work? This does not include any external JavaScript.

Reply:
missed a line🤦🏻‍♂️